source/uwp/Renderer/lib/AdaptiveDateInputRenderer.cpp

```diff
@@ -107,11 +108,33 @@ namespace AdaptiveNamespace
             }
         }
 
+        ComPtr<IAdaptiveInputElement> adapitveDateInputAsAdaptiveInput;
+        RETURN_IF_FAILED(adaptiveDateInput.As(&adapitveDateInputAsAdaptiveInput));
+
         RETURN_IF_FAILED(XamlHelpers::SetStyleFromResourceDictionary(renderContext,
                                                                      L"Adaptive.Input.Date",
                                                                      datePickerAsFrameworkElement.Get()));
 
-        XamlHelpers::AddInputValueToContext(renderContext, adaptiveCardElement, *dateInputControl);
+        ComPtr<IUIElement> datePickerAsUIElement;
+        datePicker.As(&datePickerAsUIElement);
+
+        ComPtr<IUIElement> inputLayout;
+        ComPtr<IBorder> validationBorder;
+        XamlHelpers::HandleInputLayoutAndValidation(adapitveDateInputAsAdaptiveInput.Get(),
```

Comment:
>HandleInputLayoutAndValidation [](start = 21, length = 30)

Check return values here and throughout (.As(), AddInputValue, MakeAndIntialize, CopyTo). Maybe do a pass on these types of changes and make sure we're checking returns everywhere.